packages/cosmic-swingset/lib/ag-solo/html/wallet-bridge.html

```diff
@@ -76,6 +81,14 @@
         if (obj.type === 'walletNeedDappApproval') {
           // Let them approve.
           popupWallet();
+          popped = true;
+        } else if (popped && obj.type === 'walletHaveDappApproval') {
+          ackWallet();
+          popped = false;
+        } else if (obj.type === 'walletOfferAdded') {
+          popupWallet();
+        } else if (obj.type === 'walletOfferHandled') {
+          ackWallet();
```

Comment:
hmm, why is `popped` not toggled for these? Is `popped` only about dapp approval?

Reply:
I think I'll instead make it a `pendingPopups` counter, to count `popupWallet`s that haven't got a matching `ackWallet`, so that we pulse when there's at least one `pendingPopup`.